.github/workflows/publish-snapshot.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Publish Snapshots to Gluon Nexus
         id: deploy
         run: |
-          mvn -B -ntp deploy
+          mvn -B -ntp -frta deploy
```

Comment:
Shouldn't there be a space between `-f` and `rta` ?

Reply:
it is not needed. Same with profiles `-Pandroid`, ...